src/contentScript/context.ts

```diff
@@ -45,6 +45,7 @@
 
 export function markReady(): void {
   ready = true;
+  (window as any).pbReady = true;
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/pixiebrix/pixiebrix-extension/pullRequest?prid=7684115)
```suggestion
  (window as unknown).pbReady = true;
```

Reply:
Where is this explained?

In general, would be good to formally write the types for the window interface in a file so there's one place to see what bookkeeping we've added in the PixieBrix codebase. https://mariusschulz.com/blog/declaring-global-variables-in-typescript#augmenting-the-window-interface.

Reply:
https://github.com/pixiebrix/pixiebrix-extension/pull/661#discussion_r661596020